LICENSE

```diff
@@ -1,7 +1,31 @@
-Node's license follows:
+Node.js is licensed for use as follows:
 
-====
+"""
```

Comment:
By making additive changes to the license, are we changing it? If so, it is valid? Have we consulted someone familiar with the topic on that matter?